src/codesy.coffee

```diff
@@ -5,7 +5,7 @@ codesy =
     form:
       heigth: 100
       width: 100
-    url: "https://api.codesy.io/"
+    url: "https://127.0.0.1:8443"
```

Comment:
I can create a 'dev' mode so that api.codesy.io is over written if the extension is unpacked.  This will keep from accidentally publishing the extension pointed to a local or dev url.  Thoughts?